boardswarm/src/config.rs

```diff
@@ -93,6 +94,7 @@ pub struct ModeStep {
 
 impl Config {
     pub fn from_file<P: AsRef<Path>>(path: P) -> Result<Config> {
+        info!("Trying configuration file {}", path.as_ref().display());
```

Comment:
You may wish to drop this chunk in favour of https://github.com/boardswarm/boardswarm/pull/30

Reply:
They do both serve a slightly different purpose ofcourse; This shows in the logs with config file is used not just if it failed ;) Though i'd probably be more optimistic and word it as `Loading` rather then trying but that's not a merge blocker for me